firmware/main/main.c

```diff
@@ -35,28 +35,28 @@ void app_main()
 	bool is_first_boot = nvs_init();
 	platform_init();
 
-	if (is_first_boot) {
```

Comment:
This is not mergeable as-is

Reply:
Let's merge this anyway as this step is only used in our factory and not when people flash on their own (even when their flash has been wiped).

In fact: let's switch to a micropython based solution for this instead on new badges.